hack/update-munge-docs.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+git_upstream=$(kube::util::git_upstream_remote_name)
```

Comment:
If we can get rid of another unused global, I think we're making the world a better place.  If you grep through hack/*, KUBE_GIT_UPSTREAM is set once in init and referred to only in the -munge- scripts and documented exactly nowhere.  My inclination here is to simplify and reduce.  But if you feel strongly, I can put it back.